.changeset/ninety-rings-brake.md

```diff
@@ -0,0 +1,13 @@
+---
+'@chainlink/cfbenchmarks-adapter': patch
+'@chainlink/coingecko-adapter': patch
+'@chainlink/coinmarketcap-adapter': patch
+'@chainlink/cryptocompare-adapter': patch
+'@chainlink/dxfeed-adapter': patch
+'@chainlink/moonbeam-address-list-adapter': patch
+'@chainlink/ncfx-adapter': patch
+'@chainlink/tiingo-adapter': patch
+'@chainlink/tradermade-adapter': patch
```

Comment:
Think `coinmetrics` and `dxfeed-secondary` are missing from the changeset

Reply:
Good catch. I'll add them